src/Umbraco.Web.UI.Client/src/views/packager/views/install-local.controller.js

```diff
@@ -121,23 +122,17 @@
                     },
                     installError)
                 .then(function(result) {
-
-                        if (result.postInstallationPath) {
```

Comment:
Why was this all removed??? This is very important, some packages have a custom UI that needs to be displayed after it installs, now that this logic is removed this will never happen.